spec/models/video_spec.rb

```diff
@@ -1,8 +1,35 @@
 require 'spec_helper'
 
 describe Video do 
-  it { should belong_to (:category)}
-  it { should validate_presence_of (:title)}
-  it { should validate_presence_of (:description)}
-    
-end
\ No newline at end of file
+  it { should belong_to (:category) }
+  it { should validate_presence_of (:title) }
+  it { should validate_presence_of (:description) }
+end
+
+describe "search_by_title" do
```

Comment:
When you describe the method name, first char should be either `.` or `#` if they describe a method. `.` is for class method and `#` is for instance method. - Read more here -> http://betterspecs.org/#describe